evm-contracts/src/v0.7/interfaces/TypeAndVersionInterface.sol

```diff
@@ -0,0 +1,12 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.7.0;
```

Comment:
Consider supporting more than one minor version (open range until there is a breaking change), so we can reuse it:

```suggestion
pragma solidity >0.6.0; 
```

Reply:
We tend to split our contracts by version into folders (v0.6, v0.7, v0.8). This would be a deviation from that.

I'm open to it, but we should probably have a discussion around our plans moving forward.

Reply:
For LinkToken, as an example, I'm reusing v0.6 in v0.7 so we don't duplicate all that code.

We also got feedback from ToB, in the context of Optimism audit, that the best practice would be to try to reuse as much code as we can between different targets (EVM, OVM) but also applies to different versions. The auditor was not fond of us c/p `v0.6` -> `v0.6-ovm` as most of the code is not changed.